README.md

```diff
@@ -202,10 +202,16 @@ In Native mode you can only `stealFocus` to some component to flag it as `focuse
 * **true**
 
 ##### `throttle`: integer
-Enable to throttle the function fired by the event listener.
+Enable to throttle the function fired by the event listener `keydown` and stop it by `keyup`.
```

Comment:
> ~~Enable to throttle the function fired by the event listener `keydown` and stop it by `keyup`.~~

Enable to throttle the function fired by the `keydown` event listener (canceled on `keyup` event)

(I don't really even see the point in rewriting this, as it is inherently the key-down that is throttled, so naturally key-up will cancel it? Depends on how explicit we feel we need to be I guess..)

Reply:
https://github.com/NoriginMedia/react-spatial-navigation/pull/67/commits/296cd18473ac31d0b22707be3188168646677c63

Reply:
Much better 👍 Thanks